mp/src/materialsystem/stdshaders/pbr_dx9.cpp

```diff
@@ -30,12 +33,16 @@ const Sampler_t SAMPLER_SPECULAR = SHADER_SAMPLER12;
 // Convars
 static ConVar mat_fullbright("mat_fullbright", "0", FCVAR_CHEAT);
 static ConVar mat_specular("mat_specular", "1", FCVAR_CHEAT);
+static ConVar mat_pbr_force_20b("mat_pbr_force_20b", "0");
+
+#define PBRPARALLAX
+static ConVar mat_parallax_height("mat_parallax_height", "0");
```

Comment:
This should be `FCVAR_CHEAT`. It doesn't seem to be used at the moment, can it be removed?

Reply:
yeah, forgot to remove that, just a debug CVar